lib/simp/beaker_helpers.rb

```diff
@@ -643,7 +643,7 @@ def enable_epel_on(suts)
             "https://dl.fedoraproject.org/pub/epel/epel-release-latest-#{os_maj_rel}.noarch.rpm",
           )
 
-          if os_info['name'] == 'RedHat'
+          if os_info['name'] == 'RedHat' && ENV.fetch('BEAKER_RHSM_USER', false) && ENV.fetch('BEAKER_RHSM_PASS', false)
```

Comment:
Would that be clearer as
```ruby
if os_info['name'] == 'RedHat' && ENV.key?('BEAKER_RHSM_USER') && ENV.key?('BEAKER_RHSM_PASS')
```

Reply:
Yep, silly copy/paste error

Reply:
Seems equally clear to me, but I'm not a ruby expert

Reply:
Accessing non-existent Hash key returns `nil`, so if concise == clear this could also work: 
 
```ruby
if os_info['name'] == 'RedHat' && ENV['BEAKER_RHSM_USER'] && ENV['BEAKER_RHSM_PASS'] 
```

Reply:
That failed when I was testing it, it's where I started.

Reply:
If you want to test for the key existing and also not empty, do this:
```ruby
if (os_info['name'] == 'RedHat'
  && (ENV.key?('BEAKER_RHSM_USER') && !ENV['BEAKER_RHSM_USER'].empty?)
  && (ENV.key?('BEAKER_RHSM_PASS') && !ENV['BEAKER_RHSM_PASS'].empty?))
```

Reply:
Agree with the approach proposed by @op-ct. For clarity, I would also add some parens as follows:

```ruby
if ( os_info['name'] == 'RedHat' ) && ENV['BEAKER_RHSM_USER'] && ENV['BEAKER_RHSM_PASS'] 
```